app/views/activities/index.html.slim

```diff
@@ -1,7 +1,7 @@
 h1.header
   = icon('comments-alt')
   span Activities
-  = link_to '<span>Atom Feed</span>'.html_safe, activities_path(format: :atom), class: 'atom'
+  = link_to '<span>Atom Feed</span>'.html_safe, activities_path(team_id: params[:team_id], format: :atom), class: 'atom'
```

Comment:
Exposing input parameters (the ones set in `params`) to views directly is a bit messy. In this case because `@team` is available you can just use `team_id: @team.id` here.